### DISCUSSION TITLE
Report making academic group project as open source project without concern

### DISCUSSION THREAD

**Discussion by HandsomeCoder** [community/community#28440] [Other Feature Feedback, Questions, & Ideas]
Where I can report if I found that someone from my academic group has uploaded the group project without information or taking permission and showcased complete work as their own.

**Comment by radar-blip** [comment]
Yikes, I'm not sure where but I would report them for sure, thats not even right.
Maybe Staff or someone else might chime in here. 
Good luck! 👍🏼

**Comment by airtower-luna** [comment]
I see two possible avenues here (assuming talking to the person hasn't helped), you'll have to decide what you want to do:

* If your work was published without your permission and attribution that sounds like copyright infringement. I imagine that might be complicated to determine and prove in a shared work, so if you want to pursue this direction you should probably talk to a lawyer. A [DMCA takedown request](https://docs.github.com/en/site-policy/content-removal-policies/guide-to-submitting-a-dmca-takedown-notice) is one of the actions you could pursue (again, seeking legal advice first is probably a good idea!).
* Taking credit for your work is pretty clearly academic misconduct, and I assume publishing without coordinating with the group is, too. Contact your advisor, or whoever is in charge of ensuring proper academic practice in your organization.
